### ISSUE TITLE
Can't render safe HTML tags

### ISSUE THREAD

**Issue by DrSensor** [Hashnode/user-feedback#81]
**Describe the bug**
The markdown renderer doesn't render HTML tags. (see [here](https://drsensor.hashnode.dev/testing-devblog-markdown-renderer-cjwev3ug3000oxus121dyv0jj))

**To Reproduce**
1. Copy and paste this into the editor
~~~
H<sub>2</sub>O

mc<sup>2</sup>

<details><summary>collapsible</summary>

```js
const working = burnout
```
</details>

<img align="right" alt="image positioning" src="https://camo.githubusercontent.com/b4b41004b2c9e533394e4cf4c5d66f9e2f196e44/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f6e52344c3130586c4a635365512f67697068792d646f776e73697a65642e676966">

<blockquotes>

long explanation
![images](https://camo.githubusercontent.com/b4b41004b2c9e533394e4cf4c5d66f9e2f196e44/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f6e52344c3130586c4a635365512f67697068792d646f776e73697a65642e676966)
caption

```command
$ snippet --syalalala
```
</blockquotes>

<!-- comment -->

- <details><summary>collapsible</summary>list
- <details><summary>collapsible</summary>list

<ul>
<li>multiline collapsible list</li>
<li><details><summary>Rust v1.28.0 with wasm32-uknown-unknown installed</summary>

```console
rustup default 1.28.0
rustup target add wasm32-unknown-unknown
```

</details></li>
</ul>
~~~

2. Hit `🖹 PREVIEW`

**Expected behavior**
It should be rendered akin to [this](https://gist.github.com/DrSensor/c788607a27433052742527f02e0c3ed3#file-test-md)

**Screenshots**
![image](https://user-images.githubusercontent.com/4953069/58760636-dc414680-8564-11e9-8db1-cf9680e4b847.png)

**Desktop (please complete the following information):**
 - OS: Manjaro <sub>(I think this part is irrelevant to all issues unless it's mobile)</sub>
 - Browser chrome, firefox, brave
 - Version latest

**Additional context**
Out of curiosity, what is the markdown renderer you are using?

**Comment by BolajiAyodeji** [comment]
Hi @DrSensor, thanks for this feedback.

Our markdown editor doesn't support HTML tags, all the above listed can be achieved with markdown. Please see the **HELP** section of the editor for additional help.

**Comment by minecrawler** [comment]
I disagree that everything above can be achieved with markdown (see floating images for example). Markdown is great for technical docs, but for articles, it is crippling in its vanilla form.

**Comment by BolajiAyodeji** [comment]
Hi @minecrawler, I agree with you on the floating images.

We'll look into this and see how we can improve the markdown editor.

I'll be re-opening this issue and would provide an update once I have some.

**Comment by bendtherules** [comment]
Even other things are not possible -
1. Collapsible blocks
2. Image caption
